### DISCUSSION TITLE
Question for filter strength

### DISCUSSION THREAD

**Discussion by A1b8** [isce-framework/isce2#578] [Q&A] [answered]
Dear all,
When I uesed the topsApp.py model, I have a question for the value of filter strength. How can I set the value is reasonable? Is the bigger value better or not?
I would appreciate you solving my confusion.

**Comment by fdelgadodelapuente** [comment, marked as answer]
Hi,

Filtering stregth values of 0.3 to 0.5 for small baseline interferograms are reasonable. If you increase the filter a lot to 0.9 you will see weird filtering edges that look like vertical / horizontal stripes. That's why you can't increase blindly the filter. If you are working in areas of low coherence and need to filter a lot you can increase the strength but being aware that it can introduce a lot of artifacts.

Francisco

**Reply by A1b8** [reply]
Thanks for your detailed explanation.

**Comment by EJFielding** [comment]
In areas of very high coherence, you might need to use an a very low filter strength if there are large deformation features. For the 2019 magnitude 7.2 Ridgecrest earthquake, I had to use a filter strength of 0.01 to avoid having the filter smooth over the fault ruptures.
